android/java/org/chromium/chrome/browser/BraveAdFreeCalloutDialogFragment.java

```diff
@@ -58,10 +59,12 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
 
         Button btnVideos = view.findViewById(R.id.btn_videos);
         btnVideos.setOnClickListener(button -> {
-            BraveActivity braveActivity = BraveActivity.getBraveActivity();
-            if (braveActivity != null) {
+            try {
+                BraveActivity braveActivity = BraveActivity.getBraveActivity();
                 braveActivity.focusSearchBox();
+            } catch (ActivityNotFoundException e) {
             }
```

Comment:
Would be nice to have a log in each such cases to make debugging easier.

Reply:
updated